src/octopeer-github/test/ElementEventBinding/ClickElementEventBindingTest.ts

```diff
@@ -1,24 +0,0 @@
-/// <reference path="../../main/Database/DatabaseAdaptable.ts"/>
```

Comment:
Youri said: Please explain why this test is no longer needed.
I say: this test still exists, but it is moved. Because the overlap is really small, Git doesn't say it's renamed.